pom.xml

```diff
@@ -507,8 +507,8 @@
 				<configuration>
 					<skipCheckoutIfExists>false</skipCheckoutIfExists>
 					<connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-					<scmVersionType>tag</scmVersionType>
-					<scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+					<scmVersionType>branch</scmVersionType>
+					<scmVersion>CHK-1968-improve-npg-client</scmVersion>
```

Comment:
TO be restored when [commons PR](https://github.com/pagopa/pagopa-ecommerce-commons/pull/150) is merged